pyspecProcScripts/first_level/show_pickle_table.py

```diff
@@ -0,0 +1,11 @@
+import pickle
+def show_pickle_table(pickle_file):
+    """show a pickle file with scalar variables in a
+    latex table"""
+    with open(pickle_file, "rb") as fp:
+        variables = pickle.load(fp)
+    print(r'\par\begin{tabular}{cc}')
+    print(r'\textbf{label} & \textbf{concentration}\\\hline\hline')
+    for k, v in variables.items():
+        print(f"{k} & {v} \\\\")
+    print(r'\end{tabular}\par')
```

Comment:
protein_QESR enables the user to store the concentrations with the dataset in a pickle file and this will give the pickle file in a latex table as an output

Reply:
add to the docstring that protein_QESR can take advantage of this?